hercules-ci-agent/CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
```

Comment:
I know I'll probably fail at this, but who cares.

Reply:
I'll do my best to review :)